das/dastree/dastree.go

```diff
@@ -204,7 +239,10 @@ func Content(root bytes32, oracle func(bytes32) []byte) ([]byte, error) {
 
 	preimage := []byte{}
 	for i, leaf := range leaves {
```

Comment:
Don't do this but there's an opportunity for parallel fetches across levels of the tree. If you find this function is slow I'd come back here and implement that in a future PR.

Reply:
Added a TODO there